README.md

```diff
@@ -19,6 +21,24 @@ Require it in `application.js`:
 //= require fragments
 ```
 
+### Node
+
+Yarn:
+```sh
+yarn add https://github.com/fs/fragments.js.git#commit_hash
```

Comment:
I think will be good replace custom url with appropriate from npm and release it in npm ofc.